flow-nodes/api-builder-plugin-md5/src/actions.js

```diff
@@ -1,19 +1,35 @@
 const crypto = require('crypto');
-
-function digest(req, outputs) {
-	const { data } = req.params;
+/**
+ * Action method.
+ *
+ * @param {object} params - A map of all the parameters passed from the flow.
+ * @param {object} options - The additional options provided from the flow
+ *	 engine.
+ * @param {object} options.pluginConfig - The service configuration for this
+ *	 plugin from API Builder config.pluginConfig['api-builder-plugin-pluginName']
+ * @param {object} options.logger - The API Builder logger which can be used
+ *	 to log messages to the console. When run in unit-tests, the messages are
+ *	 not logged.  If you wish to test logging, you will need to create a
+ *	 mocked logger (e.g. using `simple-mock`) and override in
+ *	 `MockRuntime.loadPlugin`.  For more information about the logger, see:
+ *	 https://docs.axway.com/bundle/API_Builder_4x_allOS_en/page/logging.html
+ * @param {*} [options.pluginContext] - The data provided by passing the
+ *	 context to `sdk.load(file, actions, { pluginContext })` in `getPlugin`
+ *	 in `index.js`.
+ * @return {*} The response value (resolves to "next" output, or if the method
+ *	 does not define "next", the first defined output).
+ */
+function digest(params, { logger }) {
+	const { data } = params;
 	if (data === undefined || data === null) {
-		return outputs.error(null, new Error('invalid argument: data'));
+		logger.error('invalid argument: data');
+		throw new Error('invalid argument: data');
 	}
-	try {
 		const hash = crypto
```

Comment:
These next few lines are at wrong indent.

Reply:
Will this be like below:
const hash = crypto
						.createHash('md5')
						.update(data)
						.digest('hex');
		return hash;

Reply:
Fixed the indentation.

Reply:
Eh, no, it's not fixed.  You made it worse:
![image](https://user-images.githubusercontent.com/871639/83658634-611cd100-a5ba-11ea-8b3d-f13c79f2cf2d.png)

Reply:
@wickedest , So is the bottom lines(createHash('md5')..) should be 1 tab ahead of the first line(const hash)?

Reply:
@deepakvk, isn't it obvious what the issue is?
![image](https://user-images.githubusercontent.com/871639/83739302-519a9800-a64d-11ea-986e-c9dc306484f8.png)

Reply:
Yes, its resolved now.